crates/services/bls_aggregation/src/bls_agg.rs

```diff
@@ -141,18 +145,26 @@ impl BlsAggregatorService {
         for (i, quorum_number) in quorum_nums.iter().enumerate() {
             quorum_threshold_percentage_map.insert(*quorum_number, quorum_threshold_percentages[i]);
         }
-
+        // todo remove unwraps and handle erorr better
         let mut operator_state_avs = self
             .avs_registry_service
             .get_operators_avs_state_at_block(task_created_block, quorum_nums.clone().into())
-            .await;
-        // throw erro if
+            .await.map_err(|e|self.logger.error(&format!("Failed to get operators state from avs registry , task_index : {}, err: {}",task_index,e), &["eigen-services-blsaggregation.bls_agg.single_task_aggregator"])).unwrap();
 
         let quorums_avs_stake = self
             .avs_registry_service
             .get_quorums_avs_state_at_block(quorum_nums.clone().into(), task_created_block)
-            .await;
-        // throw erro if != nil
+            .await
+            .map_err(|e| {
+                self.logger.error(
```

Comment:
well you're going to crash next, so this is fatal, right?

Reply:
https://github.com/Layr-Labs/eigensdk-go/blob/67787e959b727b115628a34e796df3a9ef42f646/services/bls_aggregation/blsagg.go#L247

yup the aggregator will crash

Reply:
I meant that the log entry should be a fatal not an error

Reply:
done